bin/rebuild

```diff
@@ -31,6 +32,39 @@ versiondb_head() {
   (cd "$VERSIONDB" && git rev-parse HEAD)
 }
 
+save_build_env() {
+  # saving environment information
+  filename="${LSSTSW_BUILD_DIR}/${BUILD}.env"
```

Comment:
These need to go in a subdirectory otherwise the `build/` directory is going to be extremely cluttered. I've run two tests and already feeling like this is getting out of control.

Reply:
OK, what about `build/builds/`?